packages/common/src/projects/HubProjects.ts

```diff
@@ -246,27 +249,38 @@ export async function destroyProject(
 /**
  * Search for Projects, and get IHubProject results
  *
- * Different from `searchContent` in that this returns the specific entity type
+ * Different from `hubSearch` in that this returns the IHubProjects
  *
  * @param filter
  * @param options
  * @returns
  */
 export async function searchProjects(
```

Comment:
in the other managers, his method name is just `search()`, right? If so we could (and should) introduce that here, and we could leave and deprecate `searchProjects()` - one less breaking change.

Reply:
In the Manager's the function is `search` b/c that's scoped into the Manager class... the backing module that just exports fn's needs to qualify the fn name - thus `searchProject`

Reply:
oops - I thought that was still part of the class, carry on.